packages/coinstac-ui/package.json

```diff
@@ -96,30 +95,30 @@
     "nyc": "^7.1.0",
     "react-hot-loader": "^3.1.3",
     "rimraf": "^2.6.2",
-    "sass-loader": "^6.0.7",
+    "sass-loader": "^7.1.0",
     "sinon": "^1.17.7",
     "spectron": "^3.8.0",
     "stack-chain": "^2.0.0",
-    "style-loader": "^0.16.1",
+    "style-loader": "^0.23.0",
     "tape": "^4.9.0",
-    "url-loader": "^0.5.9",
-    "webpack": "^2.7.0",
-    "webpack-dashboard": "^0.3.0",
-    "webpack-dev-server": "^2.11.2"
+    "url-loader": "^1.1.1",
+    "webpack": "^4.20.2",
+    "webpack-cli": "^3.1.2",
+    "webpack-dev-server": "^3.1.9"
   },
   "scripts": {
-    "build:webpack": "cross-env NODE_ENV=production webpack",
+    "build:webpack": "cross-env NODE_ENV=production webpack --mode=production",
```

Comment:
New webpack wants this `mode`. We could also do `mode: process.env.NODE_ENV` in webpack.config.js.